### PR TITLE
Add docs for schema - part 1.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Issue | https://github.com/yiisoft/db/issues/660

### CODE REVIEW

File: docs/en/schema/table-schema.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/**
+ * Shortcout for $db->getSchema()->getTableSchema('customer');
```
Comment (darkdef) [inline docs/en/schema/table-schema.md:71]: Shortcut

File: docs/en/schema/table-schema.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/**
+ * When `refresh` is `true`, the table schema will be re-created even if it is found in the cache.
+ * Shortcout for $db->getSchema()->getTableSchema('customer', true);
```
Comment (darkdef) [inline docs/en/schema/table-schema.md:87]: shortcut

File: docs/en/schema/table-schema.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+if ($db->getTableSchema('customer') !== null) {
+    // table exists
+    // ...your code here
+} else {
+    // table does not exist
+    // ...your code here
+}
+```
+
+> Note: If `$db->getTableSchema()` return `null`, then the table does not exist.
```
Comment (darkdef) [inline docs/en/schema/table-schema.md:115]: How about:
getTableSchema() returned object instanceof TableSchemaInterface or null if table not exists

**Review by darkdef — APPROVED** [review]

File: docs/en/schema/table-schema.md (hunk 1 above)
Comment (samdark) [inline docs/en/schema/table-schema.md:71]:
```suggestion
 * Shortcut for $db->getSchema()->getTableSchema('customer');
```

File: docs/en/schema/table-schema.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$tableSchema = $db->getTableSchema('customer');
+$tableName = $tableSchema->getName();
+```
+
+### Retrieving table full name
```
Comment (samdark) [inline docs/en/schema/table-schema.md:133]:
```suggestion
### Retrieving full table name
```

File: docs/en/schema/table-schema.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+
+You can retrieve the table full name for a given table using the `getFullName()` method of `TableSchema` class.
```
Comment (samdark) [inline docs/en/schema/table-schema.md:135]:
```suggestion
You can retrieve the full table name for a given table using the `getFullName()` method of `TableSchema` class.
```

File: docs/en/schema/table-schema.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$tableSchema = $db->getTableSchema('customer');
+$tableFullName = $tableSchema->getFullName();
+```
+
+> Note: The full name includes the schema name prefix, if any. 
+> That if the schema name is the same as the `Schema::defaultSchema`, the schema name won't be included.
```
Comment (samdark) [inline docs/en/schema/table-schema.md:150]:
```suggestion
> If the schema name is the same as the `Schema::defaultSchema`, the schema name won't be included.
```

File: docs/en/schema/table-schema.md [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$tableSchema = $db->getTableSchema('customer');
+$columns = $tableSchema->getColumns();
+```
+
+### Retrieving column
+
+You can retrieve the column metadata for a given table using the `getColumn()` method of `TableSchema` class.
```
Comment (samdark) [inline docs/en/schema/table-schema.md:219]:
```suggestion
You can retrieve the metadata for a given table column using the `getColumn()` method of `TableSchema` class.
```

**Pull request merged**